### DISCUSSION TITLE
Tier has a sdk api?

### DISCUSSION THREAD

**Discussion by Mockito-0723** [minio/minio#17563] [Q&A] [answered]
I don't find about tier api in Java sdk. How can i to create tier use api?

**Comment by klauspost** [comment, marked as answer]
To configure tiering use mc. There is no Java SDK for this - only a [Go sdk](https://github.com/minio/madmin-go).

**Reply by Mockito-0723** [reply]
How to use rest api to create tier?

**Reply by klauspost** [reply]
Other than the godoc, `mc` has implementations like [ilm-tier-add.go](https://github.com/minio/mc/blob/master/cmd/ilm-tier-add.go) and `ilm-tier-XXX` for operations.

If you need further help I suggest you look into our [subscriptions](https://min.io/pricing) where we can give more detailed help.

**Reply by Mockito-0723** [reply]
Ok,Thanks!
